blazor/rating/tooltip.md

```diff
@@ -45,11 +41,9 @@ Below example demonstrates the tooltip template of Rating.
 
 ## Tooltip customization
 
-The Rating Tooltip can be customized by using the CssClass property, which accepts custom CSS class names that defines the specific user-defined styles and themes to be applied on the Tooltip element.
-
-N> You can also explore more about [Tooltip Customization](https://blazor.syncfusion.com/documentation/tooltip/style) that shows how to customize styles and appearance of tooltip.
+You can customize the appearance of the tooltips using the `CssClass` property of the Blazor Rating component to specify custom CSS class names that define user-defined styles and themes to be applied to the tooltip element.
```

Comment:
You can customize the appearance of the tooltips using the [CssClass]() property of the Blazor Rating component and by defining the custom styles for tooltip elements like the below example.

Reply:
updated as per comments.